tests/order.py

```diff
@@ -21,27 +22,23 @@ def setUp(self) -> None:
         }
         response = self.client.post(url, data, format="json")
         json_response = json.loads(response.content)
+        self.customer = json.loads(response.content)
         self.token = json_response["token"]
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
         # Create a product category
         url = "/productcategories"
         data = {"name": "Sporting Goods"}
-        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token)
-        response = self.client.post(url, data, format="json")
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        self.category = json.loads(response.content)
```

Comment:
Did lines 33 and 34 get switched during the rebase? I think assigning self.category should come after self.client.post.

Reply:
Yes, rebase boo-boo. fixed it